components/menu/overview.md

```diff
@@ -102,6 +102,10 @@ A menu is often used to list pages, views, or sections in an application so the
 
 The Blazor Menu allows you to control its orientation and display the items horizontally or vertically. [Read more about the Blazor Menu orientation...]({%slug components/menu/orientation%})
 
+## Show and Hide Events
```

Comment:
Since the "Show" and "Hide" events are not client-side events of the Menu component, I would recommend updating the section title to "Sub-items Appearance" or "Child Items Appearance".